server/lib/config/pbench-server-default.cfg

```diff
@@ -122,6 +122,15 @@ webserver = %(default-host)s
 host-info-path-prefix = pbench-results-host-info.versioned/pbench-results-host-info.URL
 host_info_url = http://%(webserver)s/%(host-info-path-prefix)s
 
+[pbench-backup-level]
```

Comment:
Changes wil be required here: the section names should just be the script names and the logging_level should be commented out.

Reply:
then where should I define the logging-level for the scripts? in server/bin/config/pbench-server.cfg file?